fast/stages/01-resman/outputs-files.tf

```diff
@@ -16,23 +16,27 @@
 
 # tfdoc:file:description Output files persistence to local filesystem.
 
+locals {
+  outputs_location = try(pathexpand(var.outputs_location), "")
+}
+
 resource "local_file" "providers" {
   for_each        = var.outputs_location == null ? {} : local.providers
   file_permission = "0644"
-  filename        = "${pathexpand(var.outputs_location)}/providers/${each.key}-providers.tf"
-  content         = each.value
+  filename        = "${local.outputs_location}/providers/${each.key}-providers.tf"
+  content         = try(each.value, null)
```

Comment:
You have a `try()` here but not on the other `local_file`s. Is that WAI?

Reply:
that is the only one that was failing...